APP_DEVELOPMENT.md

```diff
@@ -2,26 +2,40 @@
 
 This file contains some tips and guidelines on building our front-end React Native app. Please add to it!
 
-- [Overview of directories](#overview-of-directories)
-- [Working with Figma](#working-with-figma)
-- [React Native and Expo](#react-native-and-expo)
-  - [Expo trade-offs](#expo-trade-offs)
-  - [Installing new packages](#installing-new-packages)
-  - [Using .env variables](#using-env-variables)
-  - [Expo config file](#expo-config-file)
-  - [Expo known issues](#expo-known-issues)
-- [NativeBase](#nativebase)
-  - [File locations, naming, moving and deleting](#file-locations-naming-moving-and-deleting)
-  - [Theming & approach](#theming--approach)
-  - [Dark mode](#dark-mode)
-  - [Reusable containers](#reusable-containers)
-  - [Custom components](#custom-components)
-  - [Icons](#icons)
-- [Images](#images)
-- [Logging errors and crashes](#logging-errors-and-crashes)
-  - [Seeing Bugsnag reports](#seeing-bugsnag-reports)
-  - [Logging errors to Bugsnag during development](#logging-errors-to-bugsnag-during-development)
-- [Performance issues](#performance-issues)
+- [App development](#app-development)
```

Comment:
I don't think it needs a link to the App development section as we're already here ;)